### PR TITLE
A better dropdown and auto session table generate.

### PR DESCRIPTION
**Pull request by sptutusukanta**
This PR should fix issue #8, #7.

--------------------------

This will attach advance select2 plugin to all your select elements and I've also updated the design to match your existing design.
It will also make things look alike in all browsers. I hope you'll like the implementation.

One thing I must mention though here, I setup and configured gulp but for some reason, I couldn't figure out, throwing errors when compiling the pcss. Tried too many times but failed.
That's why I've placed another css file here: `/public/css/select2.css`, if you like you may place the content of the css file and place it into `src/main.pcss` that will be automatically compiled and added to the `main.css`. With that said, you may also remove the reference to that css file in main layout file.

--------------------------

Using a Sequelize model, the automatic generation of the session table is done. Check it.

--------------------------

This is a little gift for you this **HacktoberFest 2019**. :)
I hope you'll like this.

### CODE REVIEW

File: models/session.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const sequelize = require('../util/db');
+
+const Session = sequelize.define('session', {
+    id: {
+        type: Sequelize.STRING,
+        autoIncrement: true,
+        allowNull: false,
+        primaryKey: true
+    },
+    type: {
```
Comment (prinzpiuz) [inline models/session.js]: why we need type ?

File: models/session.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        type: Sequelize.INTEGER,
+        allowNull: false
+    },
+    project: Sequelize.STRING
```
Comment (prinzpiuz) [inline models/session.js]: why we need project ?

**Comment by prinzpiuz** [comment]
for discussion regarding the same and future collaboration please join this telegram channel 
https://t.me/joinchat/FDVzKxcwfHWUyWWuTuaPHA

**Comment by sptutusukanta** [comment]
@prinzpiuz I don't understand why the old code is pushed to this PR, that's strange. Anyway, I'll push a new commit.

**Comment by sptutusukanta** [comment]
> please make separate PRs for both issues

Yeah.. sure will do from next time. I wasn't sure about that.

**Comment by sptutusukanta** [comment]
> for discussion regarding the same and future collaboration please join this telegram channel
> https://t.me/joinchat/FDVzKxcwfHWUyWWuTuaPHA

Okay, I'll look into that.

By the way, this is my first project using Postgres in node.js. I like how it worked. Though I think it could have been done in better ways, as per my experience.

**Comment by azhsetiawan** [comment]
@prinzpiuz @sptutusukanta there's a bug in add task page.

Step by step to check:
- Go to Add Task page
- Select project
- Fill other input field
- Click submit

Page load continuously because form cannot submitted.
After select __In Project__, select field for __Assigned To__ should have shown under that field.

**Comment by prinzpiuz** [comment]
@sptutusukanta can you fix it ?
i am okay with the the fix for #7

**Comment by prinzpiuz** [comment]
@sptutusukanta can push the files for #7  its good to merge

**Pull request closed without merging**